pkg/session_recording/recordings.jsx

```diff
@@ -249,7 +254,7 @@
                     <div className="row">
                         <div className="col-md-12">
                             <ol className="breadcrumb">
-                                <li><a>Session Recording</a></li>
+                                <li><a onClick={this.goBackToList}>Session Recording</a></li>
```

Comment:
This is what happens with href="/", just for info.
![screenshot from 2017-08-17 09-27-44](https://user-images.githubusercontent.com/5558501/29401275-dfbd3298-8330-11e7-8153-44d4054871de.png)

About resizable, I want to stop resizable on componentWillUnmount, but this lib is not providing disabling, will figure out something.